src/renderer/features/assets-navigation/index.ts

```diff
@@ -0,0 +1,16 @@
+import { createStore } from 'effector';
+
+import { $features } from '@/shared/config/features';
+import { createFeature } from '@/shared/effector';
+import { Paths } from '@/shared/routes';
+import { navigationTopLinks } from '@/features/app-shell';
+
+export const assetsNavigationFeature = createFeature({
+  name: 'Assets navigation',
+  input: createStore({}),
+  enable: $features.map(({ assets }) => assets),
+});
+
+assetsNavigationFeature.inject(navigationTopLinks, (items) => {
+  return items.concat({ order: 0, icon: 'asset', title: 'navigation.balancesLabel', link: Paths.ASSETS });
+});
```

Comment:
To increase readability (same for the following features).
By the way, now I see why do we have to `useI18n` in `NavItem` 😢
So until we extract i18n into effector these strings would exist
```suggestion
assetsNavigationFeature.inject(navigationTopLinks, (items) => {
  return items.concat({
    order: 0,
    icon: 'asset',
    title: 'navigation.balancesLabel',
    link: Paths.ASSETS,
  });
});
```